enhancements/console/dynamic-plugins.md

```diff
@@ -272,6 +274,21 @@ available and the user must refresh their browser to see the changes. (We won't
 refresh the page automatically to avoid possibly losing data if the user is
 entering something into a form or the YAML editor.)
 
+### Localization
+
+Dynamic plugins will need to contain their own localization files and serve them
+per requested. All dynamic plugins must use a single localization namespace
```

Comment:
I think we should stress the fact that each dynamic plugin is expected to provide all of its localized strings in a single file (using a single i18n namespace).

```
Plugins should localize strings rendered in console UI and provide a single localization
file per language, to be served as part of the plugin's assets.

For example, a plugin named `kubevirt-plugin` would have its localization files hosted
at `/locales/{language}/kubevirt-plugin.json`.

Plugins must use `plugin__` namespace prefix for all of their localized strings, e.g.
`plugin__kubevirt-plugin~Virtual Machines`. This allows console to determine whether
the namespace is related to a dynamic plugin and fetch the appropriate resource.
```

The doc already mentions that
> The console backend will proxy the plugin assets from the `Service` using the service CA bundle.

so I'd leave out the details regarding proxy URL (`/locales/resource.json?lng={{lng}}&ns={{ns}}`).